subsys/bluetooth/controller/ll_sw/nordic/lll/lll_scan_aux.c

```diff
@@ -626,11 +622,35 @@ static void abort_cb(struct lll_prepare_param *prepare_param, void *param)
 
 static void isr_done(void *param)
 {
-	struct event_done_extra *e;
+	struct lll_sync *lll;
+	uint8_t is_lll_scan;
 
 	lll_isr_status_reset();
 
-	if (!trx_cnt) {
+	if (param) {
+		lll = ull_scan_aux_lll_parent_get(param, &is_lll_scan);
+	} else {
+		lll = NULL;
+	}
+
+	/* Check if this aux scan is for periodic advertising train */
+	if (IS_ENABLED(CONFIG_BT_CTLR_SYNC_PERIODIC) && lll && !is_lll_scan) {
```

Comment:
`is_lll_scan` not initiated at this point.

Reply:
`is_lll_scan` is uninitialized when `lll = NULL`, if `lll` is NULL then `is_lll_scan` is not accessed.

Reply:
Ah, my bad, had missed the &is_lll_scan. 

In theory, since this all depends on `param`, we could move it into the `if (param)` clause as well (although not required).